app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,67 @@
+@import "application";
+.my-header {
+  background-color: $my-black;
+  height: 3.5em;
+  position: fixed;
+  width: 100%;
+}
+
+.header-menu {
+  color: $my-white;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 0.9em;
+  font-weight: 400;
+
+  ul li {
+    display: inline;
```

Comment:
Properties should be ordered color, display, padding-left, padding-right, text-decoration